crates/apub/src/api/mod.rs

```diff
@@ -87,6 +87,18 @@ fn comment_sort_type_with_default(
   )
 }
 
+/// Returns a default page fetch limit.
+/// Order is the given, then local user default, then site default.
+fn fetch_limit_with_default(
+  limit: Option<i64>,
+  local_user: Option<&LocalUser>,
+  local_site: &LocalSite,
+) -> Option<i64> {
+  limit
+    .or(local_user.and_then(|u| u.default_fetch_limit.map(i64::from)))
+    .or(local_site.default_fetch_limit.map(i64::from))
+}
+
```

Comment:
Does this mean the UI no longer needs to have a `defaultSetLimit` provided from the config?